logging_config.py

```
@@ -0,0 +1,497 @@
+    def wrapper(*args, **kwargs):
+        logger = get_logger(func.__module__)
+        
+        # Get request context
+        request_id = request_id_var.get()
+        correlation_id = correlation_id_var.get()
+        
+        start_time = time.time()
+        
+        # Log function entry
+        logger.debug(
+            f"Entering {func.__name__}",
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)